pages/index.tsx

```diff
@@ -118,21 +126,28 @@ function renderPresenter(
  *
  */
 export const Home: React.FC<HomeProps> = ({ sectionCollection }) => {
-  const contentCollection = sectionCollection?.items?.flatMap(
-    (item) => item?.contentCollection?.items ?? []
-  )
-
   return (
     <LayoutFramework navigation={renderNavigation(sectionCollection)}>
-      {contentCollection.map(
-        ({ __typename, ...fields }: SectionContentItem) => {
-          const { title: key } = fields
-
-          return React.cloneElement(
-            renderPresenter(__typename as keyof SectionContentItem, fields),
-            {
-              key,
-            }
+      {sectionCollection?.items.map(
+        ({ title, contentCollection: { items } }) => {
+          return (
+            <Section key={title}>
+              {items.map(
```

Comment:
Could anything be done to make this more readable? Perhaps abstracting out the `mapItem` function..?